### PR TITLE
Closes #3391: Hyperlink Land Acknowledgement text in the Global Footer

### LINKED ISSUE #3391: Hyperlink Land Acknowledgement text in the Global Footer
## Problem/Motivation

Frank Camp in the Marketing & Communications office has requested we hyperlink the Land Acknowledgement text in the global footer. 

Can a hyperlink please be added to the 'the University of Arizona is on the land and territories of Indigenous peoples.' portion of the Land Acknowledgement, linking out to [https://storymaps.arcgis.com/stories/913da25f6c3d46658690c3800bfef48e ](https://storymaps.arcgis.com/stories/913da25f6c3d46658690c3800bfef48e )? 

Here is a screenshot of the Land Acknowledgement placement in the global footer: [https://app.screencast.com/kDck0dVEAFTOA ](https://app.screencast.com/kDck0dVEAFTOA 
)
Thank you! Catherine Gustafson 

### Update 11-18-2024:

A ServiceNow request has been received as a follow-up to this issue. See [SCTASK0058256](https://uarizona.service-now.com/nav_to.do?uri=sc_task.do%3Fsys_id=e90486cb977916108a15fae3a253af00%26sysparm_stack=sc_task_list.do%3Fsysparm_query=active=true).

> For the global footer please update the following text to be a hyperlink to https://www.arizona.edu/university-arizona-land-acknowledgment
> 
> "the University of Arizona is on the land and territories of Indigenous peoples"
> 
> Screenshot: https://app.screencast.com/Qj9wi2Sh1ppeI

## Proposed resolution

Chris Green says:

FYI, the code to change is here, [https://github.com/az-digital/az_quickstart/blob/1b175ea19531325f7585d585556d2ce751867cf7/themes/custom/az_barrio/az_barrio.theme#L133](https://github.com/az-digital/az_quickstart/blob/1b175ea19531325f7585d585556d2ce751867cf7/themes/custom/az_barrio/az_barrio.theme#L133)

I think this can be labeled good first issue

### PR DESCRIPTION
**Pull request by danahertzberg**
(Updated 11-18-2024)

## Description
Adds link to Land Acknowledgement in the Global Footer

## Related issues
Closes #3391 

## How to test
Ensure footer link is on "the University of Arizona is on the land and territories of Indigenous peoples" and opens correct link (https://www.arizona.edu/university-arizona-land-acknowledgment)

Probo build: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-3400.probo.build

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [x] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by joshuasosa** [comment]
I think it'd be better if the link went to an arizona.edu domain and if the page was set up as a set of pages. The arcgis page is fine, it works, but it's a _very long page_ which doesn't lend itself to reading digestibility. According to the page's footer, it's put on by James E. Rogers College of Law at UArizona. Being that this statement and link appears at the bottom of every single webpage that uses AZ Quickstart, I think it's important enough and significant enough to warrant its own presence on arizona.edu instead of a third-party site.

**Comment by danahertzberg** [comment]
Thank you @joshuasosa! I will share the feedback with the marketing team.

**Comment by danahertzberg** [comment]
> **Henry Greenberg**
> After talking with Sabre she would like to hold off doing this. Considering that this link would be on every Quickstart site she would prefer that this content live on a [Arizona.edu](http://arizona.edu/) website ( Pantheon) under CWS control. So that we can shut it down should the need arise. I will communicate this to the CWS team as well. This request started with Frank and she will be talking to him about it directly. Stay tuned…

**Comment by danahertzberg** [comment]
Closing until we have confirmed next steps

**Comment by bberndt-uaz** [comment]
Reopening this pull request since we have received an updated URL to use for this land acknowledgment link: https://www.arizona.edu/university-arizona-land-acknowledgment.

File: themes/custom/az_barrio/az_barrio.theme [hunk 1]
```diff
@@ -130,7 +130,7 @@ function az_barrio_preprocess_page(&$variables, $hook) {
...
   $land_acknowledgment = '';
   if (theme_get_setting('land_acknowledgment')) {
-    $land_acknowledgment = '<p class="font-weight-light"><em>We respectfully acknowledge the University of Arizona is on the land and territories of Indigenous peoples. Today, Arizona is home to 22 federally recognized tribes, with Tucson being home to the O’odham and the Yaqui. Committed to diversity and inclusion, the University strives to build sustainable relationships with sovereign Native Nations and Indigenous communities through education offerings, partnerships, and community service.</em></p><hr>';
+    $land_acknowledgment = '<p class="font-weight-light"><em>We respectfully acknowledge <a href="https://storymaps.arcgis.com/stories/913da25f6c3d46658690c3800bfef48e">the University of Arizona is on the land and territories of Indigenous peoples</a>. Today, Arizona is home to 22 federally recognized tribes, with Tucson being home to the O’odham and the Yaqui. Committed to diversity and inclusion, the University strives to build sustainable relationships with sovereign Native Nations and Indigenous communities through education offerings, partnerships, and community service.</em></p><hr>';
```
Comment (bberndt-uaz) [inline themes/custom/az_barrio/az_barrio.theme:133]:
```suggestion
    $land_acknowledgment = '<p class="font-weight-light"><em>We respectfully acknowledge <a href="https://www.arizona.edu/university-arizona-land-acknowledgment">the University of Arizona is on the land and territories of Indigenous peoples</a>. Today, Arizona is home to 22 federally recognized tribes, with Tucson being home to the O’odham and the Yaqui. Committed to diversity and inclusion, the University strives to build sustainable relationships with sovereign Native Nations and Indigenous communities through education offerings, partnerships, and community service.</em></p><hr>';
```

**Review by bberndt-uaz — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

**Pull request merged**